lib/omnibus/health_check.rb

```diff
@@ -35,6 +35,7 @@ class HealthCheck
       /libutil\.so/,
       /linux-vdso.+/,
       /linux-gate\.so/,
+      /libffi\.so/,
```

Comment:
Building on new platforms (specifically for Cisco) the healthcheck for
the chef suite was failing because libffi was not on the allowed list.
This corrected that.

On 07/23/2015 02:09 PM, Seth Chisamore wrote:
>
> In lib/omnibus/health_check.rb
> <https://github.com/chef/omnibus/pull/515#discussion_r35359464>:
>
> > @@ -35,6 +35,7 @@ class HealthCheck
> >        /libutil\.so/,
> >        /linux-vdso.+/,
> >        /linux-gate\.so/,
> > +      /libffi\.so/,
>
> Why are we whitelisting this? libFFI is not a system library.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chef/omnibus/pull/515/files#r35359464>.
>